huggingface/pytorch/training/docker/2.3/py3/cu121/Dockerfile.gpu

```diff
@@ -0,0 +1,63 @@
+# https://github.com/aws/deep-learning-containers/blob/master/available_images.md
+# refer to the above page to pull latest Pytorch image
+
+# docker image region us-west-2
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:2.3.0-gpu-py311-cu121-ubuntu20.04-sagemaker
```

Comment:
ubuntu 20 is reaching EOL, can we use a later version? 
https://ubuntu.com/about/release-cycle

Reply:
Hi @e-davidson, we will use ubuntu 22.04 from PyTorch 2.4 DLCs.